### PR TITLE
Adds Multiselect to the Table Widget

### PR DESCRIPTION
**Pull request by anmoljagetia**
@GordonSmith @mlzummo Please review!
Fixes GH-427
Signed-off-by: Anmol Jagetia <anmoljagetia@gmail.com>

### CODE REVIEW

File: src/other/Table.css [hunk 1]
```diff
@@ -26,8 +26,13 @@
     color:black;
 }
 
+.other_Table .selected {
```
Comment (GordonSmith) [inline src/other/Table.css]: I would change this to:  .other_Table tr.selected

File: src/other/Table.js [hunk 2]
```diff
@@ -137,8 +138,35 @@
...
+                    }
+                };
+                context._selection.click(newObj, d3.event);
+            })
         ;
 
+        rows
+            .attr("class", function (d) {
+                // alert(this);
+                // alert(d);
```
Comment (GordonSmith) [inline src/other/Table.js]: Remove commented code

File: src/other/Table.js [hunk 3]
```diff
@@ -201,5 +228,27 @@
...
+        if (!arguments.length) {
+            var t = [];
+            for (var i = 0; i < this._selection.get().length; i++) {
+                t[i] = this._selection.get()[i]._id;
+            }
+            return t;
+        } else {
+            var t = [];
+            for (var i = 0; i < _.length; i++) {
+                t[i] = {
```
Comment (mlzummo) [inline src/other/Table.js]: i'd do the ind. on this a bit dif:
t[i] = {
	"_id": _[i],
	element: function() {
	      return d3.select(null);
	}
}

File: src/other/Table.js [hunk 4]
```diff
@@ -137,8 +138,35 @@
...
+            })
         ;
 
+        rows
+            .attr("class", function (d) {
+                // alert(this);
+                // alert(d);
+                var domNode = this;
+                var newObj = {
+                  "_id": d,
```
Comment (mlzummo) [inline src/other/Table.js]: fix ind.? like below?

**Comment by GordonSmith** [comment]
Code looks fine.  From a learning point of view, take a look at: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map  It exists specifically to transform an array of objects to anouther array shape...

**Comment by GordonSmith** [comment]
Looks good.

**Pull request merged**